runtime-modules/content-directory/src/lib.rs

```diff
@@ -1091,19 +1145,24 @@ decl_module! {
             let property_value_vector =
                 entity.ensure_property_value_is_vec(in_class_schema_property_id)?;
 
-            //
-            // == MUTATION SAFE ==
-            //
+            let class_id = entity.get_class_id();
 
             // Calculate side effects for clear_property_vector operation, based on property_value_vector provided and its respective property.
             let entities_inbound_rcs_delta = Self::make_side_effects_for_clear_property_vector_operation(&property_value_vector, property);
 
-            // Decrease reference counters of involved entities (if some)
-            Self::update_entities_rcs(&entities_inbound_rcs_delta);
-
             // Clear property_value_vector.
             let empty_property_value_vector = Self::clear_property_vector(property_value_vector);
 
+            // Ensure provided `Property` with `unique` flag set is `unique` on `Class` level
+            Self::ensure_property_unique_option_satisfied(class_id, in_class_schema_property_id, &empty_property_value_vector.simplify())?;
```

Comment:
This is basically a check that there is no empty vector for this, possible, unique property. Lets update the comment to say exactly that.

Reply:
Done